src/app/homepage/registration/registration.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {MatDialogRef} from "@angular/material/dialog";
+import {AuthService} from "../../auth/auth.service";
+
+@Component({
+  selector: 'app-registration',
+  templateUrl: './registration.component.html',
+  styleUrls: ['./registration.component.css']
+})
+export class RegistrationComponent {
+
+  constructor(
+    public dialogRef: MatDialogRef<RegistrationComponent>,
+    public authenticationService: AuthService) {
```

Comment:
If you add here the method of the html then it would be also obviouse why you need that authenticationService. Additionally Injected Services should be private.